File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        String[] answer = new String[quiz.length];
        
        for(int i=0; i< quiz.length; i++){
            String[] s = quiz[i].split(" ");
            int n = Integer.parseInt(s[0]);
            if(s[1].equals("+"))
                n += Integer.parseInt(s[2]);
            else
                n -= Integer.parseInt(s[2]);
            
            if(s[4].equals(String.valueOf(n)))
                answer[i] = "O";
            else 
                answer[i] = "X";
        }//for
        
        return answer;
    }
}

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    for i in quiz:
        li = i.split(' ')
        num = int(li[0])+int(li[2]) if li[1]=='+' else int(li[0])-int(li[2])
        answer.append('O' if num==int(li[4]) else 'X')
    return answer